apps/production/src/task-manager/task-manager.service.ts

```diff
@@ -983,4 +979,24 @@ export class TaskManagerService {
 
     return totalInstalls / extensions.length
   }
+
+  @Cron(CronExpression.EVERY_5_SECONDS)
+  async sendTelegramMessages() {
+    try {
+      const messages = await this.telegramService.getMessages()
+
+      messages.forEach(async message => {
+        await this.telegramService.sendMessage(
+          message.id,
+          message.chatId,
+          message.text,
+          message.extra,
+        )
```

Comment:
Why not to use the following instead:
```suggestion
      const promises = _map(messages, (async (message) => {
        await this.telegramService.sendMessage(
          message.id,
          message.chatId,
          message.text,
          message.extra,
        )
      })

      await Promise.all(promises)
```

Reply:
The first option, using `forEach` and awaiting each `this.telegramService.sendMessage` call, guarantees sequential message sending, ensuring a more reliable and predictable execution flow.